.gitlab-ci.yml

```diff
@@ -126,77 +126,42 @@ OPENEBS-STORAGE-POLICIES:
     - chmod 755 ./stages/openebs-install/openebs-storage-policies
     - ./stages/openebs-install/openebs-storage-policies
 
-tcid-gaau01-gui-auth:
+TCID-GUI-AUTH:
   image: harshshekhar15/gitlab-job:v5
   stage: DIRECTOR-GUI
   dependencies:
     - TCID-DIR-INSTALL-ON-LOCAL-HP
   script:
-    - chmod 755 ./stages/director-gui/tcid-gaau01-gui-auth
-    - ./stages/director-gui/tcid-gaau01-gui-auth
+    - chmod 755 ./stages/director-gui/gui-auth-check
```

Comment:
@rohan2794 let's not use `-check` in the name of the scripts as this an E2E pipeline and almost all job is a testcase.